### DISCUSSION TITLE
Add  information of base Firefox version to about page

### DISCUSSION THREAD

**Discussion by Yuki2718** [Floorp-Projects/Floorp#982] [Feature Request]
Currently only the version of Floorp is shown.

<details>
<summary>Screenshot</summary>

![about](https://github.com/Floorp-Projects/Floorp/assets/58900598/f5b51909-060e-4ae7-9304-e3b9b1f4f862)

</details>

If you want to know the Firefox version it is based on, you have to go through `about:config` or something. Someone even [asked](https://github.com/Floorp-Projects/Floorp/discussions/620) the version in Discussion. Brave shows its base Chromium verison like `version: 1.63.165 Chromium: 122.0.6261.94` and this is convenient for other developers when they see reports with browser version if there are known issues on certain version of Chromium.

**Comment by CutterKnife** [comment]
`about:preferences` で確認できますが、確かに "Floorp について" にあれば便利かもしれませんね。

**Comment by regularentropy** [comment]
![image](https://github.com/Floorp-Projects/Floorp/assets/89523758/6aa83b65-2d63-43a7-9f3e-cc9829185359)
You can check it in about:preferences
